.github/workflows/cmake_ctest.yml

```diff
@@ -119,14 +119,31 @@ jobs:
       with:
         submodules: recursive # Get JUCE populated
 
-    - name: Setup Environment Variables
+    - name: Setup Common Environment Variables
       shell: bash
       run: |
         VERSION=$(cat VERSION)
         echo "ARTIFACTS_PATH=${{ env.BUILD_DIR }}/${{ env.PROJECT_NAME }}_artefacts/${{ env.BUILD_TYPE }}" >> $GITHUB_ENV
-        echo "STANDALONE_PATH=${{ env.PROJECT_NAME }}_artefacts/${{ env.BUILD_TYPE }}/${{ env.PROJECT_NAME }}.app" >> $GITHUB_ENV
         echo "PRODUCT_NAME=${{ env.PROJECT_NAME }}-$VERSION-${{ matrix.name }}" >> $GITHUB_ENV
 
+    - name: Setup Environment Variables for macOS
+      if: matrix.name == 'macOS'
+      shell: bash
+      run: |
+        echo "STANDALONE_PATH=${{ env.PROJECT_NAME }}_artefacts/${{ env.BUILD_TYPE }}/${{ env.PROJECT_NAME }}.app" >> $GITHUB_ENV
+
+    - name: Setup Environment Variables for Windows
+      if: matrix.name == 'Windows'
+      shell: bash
+      run: |
+        echo "STANDALONE_PATH=${{ env.PROJECT_NAME }}_artefacts/${{ env.BUILD_TYPE }}/${{ env.PROJECT_NAME }}.exe" >> $GITHUB_ENV
+
+    - name: Setup Environment Variables for Linux
+      if: matrix.name == 'Linux'
+      shell: bash
+      run: |
+        echo "STANDALONE_PATH=${{ env.PROJECT_NAME }}_artefacts/${{ env.BUILD_TYPE }}/${{ env.PROJECT_NAME }}" >> $GITHUB_ENV
```

Comment:
Should there be any sort of extension for Linux? I know functionally it shouldn't matter.

Reply:
I think in Linux it is most common to not have any extension for executable files, so maybe this is fine.